cockatrice/src/chatview.cpp

```diff
@@ -130,7 +130,7 @@ void ChatView::appendMessage(QString message, RoomMessageTypeFlags messageType,
     lastSender = sender;
     
     // timestamp
-    if (showTimestamps && !sameSender && !sender.isEmpty()) {
+    if (showTimestamps && (!sameSender || sender == "Servatrice") && !sender.isEmpty()) {
```

Comment:
Its up to you.  I don't think its lowered else were and since the server sends the message through  a function that pre-populates that field it should be consistent.